### ISSUE TITLE
[MAS1.4.3] Luminosity Contrast Ratio of text in grey present in Log panel is less than required 4.5:1.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-Emulator#703]
**Actual Result:**
Luminosity contrast ratio of grey text present in Log panel is 4.2:1.

**Expected Result:**
Luminosity contrast ratio of grey text present in Log panel should be more than equal to 4.5:1.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Click on open bot and choose the .bot file that is available in the bot folder.
3. Verify that live chat tab is opened.
4. Notice Log panel present in bottom right of screen.
5. Notice grey text present in this panel.
6. Open 'Colour Contrast Analyser' application.
7. Select Foreground and Background color of mentioned text.
8. Check value of 'Luminosity Contrast Ratio'.

**User Impact:**
Users with low vision or color blindness needs that difference between contrast of text is always high.

**Test Environment:**
OS: Windows 10
OS Build: 17134.167
OS Version: 1803
Browser: Edge

**Impacted MAS Link:**
[MAS 1.4.3 – Contrast (Minimum) (17)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6})

**Attachment:**
![mas1 4 3_botframeworkemulator_log_greytext](https://user-images.githubusercontent.com/41668935/43193404-c6ac591e-8fb4-11e8-883b-7b1a9cb74032.png)

**Comment by corinagum** [comment]
@mewa1024 Could you let me know what this color should be updated to? Currently it's #cccccc

**Comment by mewa1024** [comment]
@corinagum Can we use #9E9E9E? 
@justinwilaby We don't use #9E9E9E anywhere else, but maybe to avoid adding a new color, we could replace #A6A6A6 with #9E9E9E?

**Comment by corinagum** [comment]
Related to issue [725](https://github.com/Microsoft/BotFramework-Emulator/issues/725);
